freqtrade/freqtradebot.py

```diff
@@ -619,7 +636,14 @@ def handle_trade(self, trade: Trade) -> bool:
         return False
 
     def check_sell(self, trade: Trade, sell_rate: float, buy: bool, sell: bool) -> bool:
-        should_sell = self.strategy.should_sell(trade, sell_rate, datetime.utcnow(), buy, sell)
+        if (self.config['edge']['enabled']):
+            stoploss = self.edge.stoploss(trade.pair)
+            should_sell = \
+                self.strategy.should_sell(trade, sell_rate, datetime.utcnow(), buy, sell, stoploss)
```

Comment:
i think we should use implicit line breaks (within parenteses) where possible - as encouraged by PEP8 ... ;)